rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -316,58 +378,21 @@ def _check_config_parameters(self) -> None:
             self.component_config
         )
 
-    # package safety checks
-    @classmethod
-    def required_packages(cls) -> List[Text]:
+    @staticmethod
+    def required_packages() -> List[Text]:
+        """Any extra python dependencies required for this component to run."""
         return ["tensorflow"]
```

Comment:
funny that `TEDPolicy` doesn't do this. Might be worth adding there actually for consistency.

Reply:
I guess it's not actually, as it's a dependency of rasa. Shall I remove it?